corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditorMain.java

```diff
@@ -189,6 +193,32 @@ public static void main(String[] args) {
                             "Key To Delete is Null.");
                     browser.deleteRecord(namespace, tableName, keyToDelete);
                     break;
+                case addRecord:
+                    String keyToAdd = null;
+                    String recordToAdd = null;
+                    String metadataToAdd = null;
+                    if (opts.get("--keyToAdd") != null) {
+                        keyToAdd = String.valueOf(opts.get("--keyToAdd"));
+                    }
+                    if (opts.get("--newRecord") != null) {
```

Comment:
This recordToAdd is later combined with newMetadata to form a dynamicRecord. That seems a bit confusing.
We might need to make the terms we use consistent -- <key, value, metadata> is a set of concepts and here it really represents the `value`.

Reply:
do you mean the name 'newRecord' here is confusing because internally, DynamicRecord represents a <value, metadata>?

Reply:
Yes that's my confusion. I thought a user should provide a value and metadata here but actually this field is just for value.

Reply:
ok, I can rename it.  The reason for keeping it as 'newRecord' was to keep it consistent with 'editTable' where 'newRecord' just means the 'value'.  But 'value' is easier to understand both internally and for other users.

I will just change it for this operation right now and not 'editTable'.